racetime/urls.py

```diff
@@ -31,13 +31,10 @@
         path('userinfo', views.OAuthUserInfo.as_view(), name='oauth2_userinfo'),
     ])),
 
-    path('autocomplete/', include([
-        path('user', views.AutocompleteUser.as_view(), name='autocomplete_user'),
-    ])),
-
```

Comment:
I think this path should be left in to avoid a BC break. No real reason to remove it.